docs/administration/protect-sensitive-configuration.md

```diff
@@ -0,0 +1,36 @@
+# Protect sensitive configuration
+
+Some administrative actions and configuration modifications via the user interface can lead to data loss during ingestion, loss of data visibility for users, disruption of predefined automations, slowness
+, etc.
+To protect platforms from uncontrolled actions and make administrators' tasks easier, it is possible to restrict the modification of certain configurations to specific users.
```

Comment:
Should we say since 6.4.x ?

Reply:
documentation is already versioned. If you're running 6.3.X you should seek the corresponding documentation.
I'm not fond of adding such precision everywhere (with time, it will become flooded with 'since X.X' that have not much utility.

my two cents...